docs/_tutorials/orderofinterpretation.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: tutorials
+permalink: /tutorials/orderofinterpretation/
+title: Order of interpretation
+---
+
+Jekyll's main job is to convert your raw text files into a static website. It does this by rendering Liquid, Markdown, and other transforms as it generates the static HTML output.
+
+In this conversion process, it's important to understand Jekyll's order of interpretation. By "order of interpretation," we mean what gets rendered, in what order, and what rules get applied in converting content.
+
+If an element isn't converting, you can troubleshoot the problem by analyzing the order of interpretation.
+
+## Order of interpretations
+
+Jekyll converts your site in the following order:
+
+1. **Site variables**. Jekyll looks across your files and populates [site variables]({% link _docs/variables.md %}), such as `site`, `page`, `post`, and collection objects. (From these objects, Jekyll determines the values for permalinks, tags, categories, and other details.)
+
+2. **Liquid**. Jekyll processes any Liquid formatting in pages that contain [front matter]({% link _docs/frontmatter.md %}). All Liquid tags, such as `include`, `highlight`, or `assign` tags, are rendered. (Anything in Jekyll with `{% raw %}{{ }}{% endraw %}` curly braces or `{% raw %}{% %}{% endraw %}` is usually a Liquid filter or tag.)
```

Comment:
Instead of saying
```
(Anything in Jekyll with {% raw %}{{ }}{% endraw %} curly braces or {% raw %}{% %}{% endraw %} is usually a
Liquid filter or tag.)
```
and generalising, I feel its better to inform what's what.. i.e. : 

---
#### Identify Liquid by the presence of curly brackets / braces:
  - **Liguid Tags:** They start with a `{%` and end with a `%}`. e.g. `{% raw %}`, `{% highlight %}`, `{% css -- %}`, `{% seo %}`. Tags can define blocks or can be inline. Block-defining tags will additionally come with a corresponding end-tag 
i.e `{% endraw %}`, `{% endhighlight %}` 
  - **Liquid Variables:** They start and end with double-braces. e.g. `{{ site.myvariable }}` , `{{ content }}`
  - **Liquid Filters:** They start with a pipe character (`|`) and can only be used within **Liquid Variables** after the variable string. e.g `relative_url` filter in `{{ "css/main.css" | relative_url }}`

Learn more about Liquid by referring the official documentation `[LINK]`

---